docs/en/05_Contributing/01_Code.md

```diff
@@ -59,7 +59,43 @@ We ask for this so that the ownership in the license is clear and unambiguous, a
 
 The core team is then responsible for reviewing patches and deciding if they will make it into core.  If
-there are any problems they will follow up with you, so please ensure they have a way to contact you! 
+there are any problems they will follow up with you, so please ensure they have a way to contact you!
+
+### The Pull Request Process
+
+Once your pull request is issued, it's not the end of the road. A [core committer](/contributing/core_committers/) will most likely have some questions for you and may ask you to make some changes depending on discussions you have.
+If you've been naughty and not adhered to the coding conventions, expect a few requests to make changes so your code is in-line.
+
+If your change is particularly significant, it may be referred to the [mailing list](https://groups.google.com/forum/#!forum/silverstripe-dev) for further community discussion.
+
+A core committer will also "label" your PR using the labels defined in GitHub, these are to correctly classify and help find your work at a later date.
+
+#### GitHub Labels
+
+The current GitHub labels are grouped into 5 sections:
+
+ 1. Changes - These are designed to signal what kind of change they are and how they fit into the [Semantic Versioning](http://semver.org/) schema
+ 2. Impact - What impact does this bug/issue/fix have, does it break a feature completely, is it just a side effect or is it trivial and not a bit problem (but a bit annoying)
+ 3. Effort - How much effort is required to fix this issue?
+ 4. Type - What aspect of the system the PR/issue covers
+ 5. Feedback - Are we waiting on feedback, if so who from? Typically used for issues that are likely to take a while to have feedback given
+
+| Label | Purpose |
+| ----- | ------- |
+| change/major | A change to go against the next major release (eg: 4.0) |
+| change/minor | A change to go against the next minor release (eg: 3.x) |
+| change/patch | A change to go against the next patch release (eg: 3.1.x) |
+| impact/critical | This is a bug which breaks functionality in a way that can not be worked around and must be fixed |
+| impact/high | This is a bug which breaks functionality but can be mitigated by other code changes |
+| impact/medium | This is a bug which has unexpected behaviour but does not break functionality |
+| impact/low | This is a bug which is a nuisance but doesn't break any functionality (typos, etc) |
+| effort/easy | This is a bug that someone with limited SilverStripe experience could resolve |
+| effort/medium | This is a bug that someone with a good understanding of SilverStripe could resolve |
+| effort/hard | This is a bug that only an expert with SilverStripe could resolve |
+| type/docs | This is a docs change |
+| type/frontend | This is a change to front-end (CSS, HTML, etc) |
+| feedback-required/core-team | This issue is awaiting feedback from core team members |
```

Comment:
I suppose that it's more for things that need group consideration, rather than a "default state" where some kind of review is needed.

Really, I only invisage them being used on PRs or issues with a long lifetime or where feedback needs to be collated, rather than switching them everytime someone replies